test/inductor/test_decompose_mem_bound_mm.py

```diff
@@ -71,31 +73,46 @@ def forward(
 )
 @instantiate_parametrized_tests
 class TestDecomposeMemMM(TestCase):
-    def compare_dict_tensors(self, ref_dict, res_dict, rtol=1e-3, atol=1e-3):
+    def __init__(self, method_name='runTest', methodName='runTest'):
+        super().__init__(method_name, methodName)
+        self.atol = 1e-3
+        self.rtol = 1e-3
+
+    def setup_tolerance(self, rtol=None, atol=None):
+        if rtol is None:
+            rtol = self.rtol
+        if atol is None:
+            atol = self.rtol
```

Comment:
How does this work?  rtol and atol local to the function.  Modifying them here should not affect call sites like line 88 below.  Right?